src/integrationTest/resources/tests/dailylist/DailyListServiceTest/insert1_duplicate_ok/DailyListRequest.json

```diff
@@ -0,0 +1,89 @@
+{
+  "document_id": {
+    "document_name": "DailyList_457_20210219174938.xml",
+    "unique_id": "CSDDL1613756980160",
+    "document_type": "DL",
+    "time_stamp": "2021-02-19T17:49:38.391+01:00"
```

Comment:
I thought we were expecting to receive only UTC date-time strings?
`OffsetDateTime` should handle this fine however, it is just a bit confusing seeing different timezones in the same request :)